client/src/pages/Requests/requests.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import axios from 'axios';
+
+class Request extends React.Component {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  constructor(props: {} | Readonly<{}>) {
+    super(props);
+    this.state = {
+      requestInfo: null,
+      error: '',
+    };
+  }
+
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+  getRequestInfo = () => {
+    axios
+      .get('/request')
+      // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
While this library is useful, we have been utilising another library in this project called `fetch`. Would have been nice to see that being used for consistency instead of adding a new library to the project.